### ISSUE TITLE
[Feature] Scheduler Timeline Month allowing arbitrary start & end date.

### ISSUE THREAD

**Issue by cirdog** [syncfusion/ej2-javascript-ui-controls#57]
Dear Dev Team,

We have evaluated the Scheduler but noticed an innate limitation that prohibited a use case that we required, that is allowing "**arbitrary start & end date**" for Timeline Month view.

To simply put, our use case required the UI to show a "**moving window**" of a month, always centered at today. But the currently limitation would totally prohibit this approach & we'll end up with say Today is Aug 30th so we have only 1 day ahead & all days in the past.

Would you please consider allowing this? Then Timeline Month would be **so much more usefu**l. Thanks very much for your consideration.

**Comment by sellakumark** [comment]
Dear Customer, 
 
Syncfusion greetings. 
 
Currently it is not possible to render custom range of dates in timeline schedular and for the same we have already logged the feature request which can be tracked from the following link. It will be implemented in any one of our upcoming main release. 
https://www.syncfusion.com/feedback/5075/custom-view-support-on-scheduler-with-user-defined-date-range  
 
Mean time we suggest you to use interval option of view to customize the TimelineMonth and for the same we have prepared the below sample. 
https://stackblitz.com/edit/rwc6ag-ey9vv4?file=index.ts   
 
Also we could reproduce the reported issue of “scroll bar not moving to visualize the today” and logged the below bug report and fix will be included in our upcoming Volume 3 release which is expected to roll out by end of September.We would appreciate your valuable patience until then. 
https://www.syncfusion.com/feedback/8542/scroll-not-work-properly-in-timelinemonth-view 

Regards,
Sella
